README.md

```diff
@@ -1,14 +1,66 @@
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 
-![Ui](docs/images/Ui.png)
```

Comment:
As per [instructions](https://nus-tic4002-ay2122s2.github.io/website/schedule/week6/project.html), we should keep this